basilisk/src/darcsit/apply.sh

```
@@ -0,0 +1,11 @@
+# If any applied patches adds or removes files, the Makefile.tests in
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)